aries_cloudagent/multitenant/admin/routes.py

```diff
@@ -444,22 +446,22 @@ async def wallet_remove(request: web.BaseRequest):
         body = await request.json()
         wallet_key = body.get("wallet_key")
 
-    async with context.session() as session:
-        try:
+    try:
+        async with context.session() as session:
             multitenant_mgr = session.inject(BaseMultitenantManager)
             wallet_record = await WalletRecord.retrieve_by_id(session, wallet_id)
```

Comment:
same as above

Reply:
Fixed in this MR https://github.com/globalid/aries-cloudagent-python/pull/20/commits/89fc9e5f5a0f4f3ae82c141932309b08f9c3aced